### DISCUSSION TITLE
Will Esbonio be published on open-vsx.org?

### DISCUSSION THREAD

**Discussion by rbastens** [swyddfa/esbonio#550] [Ideas]
Currently Esbonio is not available on [Open VSX](https://open-vsx.org/). Is it possible to publish Esbonio on Open VSX?

**Comment by alcarney** [comment]
Ooh, I hadn't come across this before!
Definitely open to the idea, I'm all for making esbonio more widely available :smile:

**Reply by rbastens** [reply]
Thanks, that would be great!

**Comment by alcarney** [comment]
If you are still interested, you should now be able to install `v0.93.2` from the [Open VSX marketplace](https://open-vsx.org/extension/swyddfa/esbonio)! Feel free to open an issue if there are any problems :)

**Reply by rbastens** [reply]
Thanks, I'm definitely still interested!
